docs/release/fundamentals/queries-and-expressions.ipynb

```diff
@@ -0,0 +1,2023 @@
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<style type=\"text/css\">\n",
+       "#T_6f30e th {\n",
+       "  text-align: center;\n",
+       "}\n",
+       "#T_6f30e_row0_col0, #T_6f30e_row0_col1, #T_6f30e_row0_col2, #T_6f30e_row1_col0, #T_6f30e_row1_col1, #T_6f30e_row1_col2, #T_6f30e_row2_col0, #T_6f30e_row2_col1, #T_6f30e_row2_col2 {\n",
+       "  white-space: pre-wrap;\n",
+       "  text-align: left;\n",
+       "}\n",
+       "</style>\n",
+       "<table id=\"T_6f30e\">\n",
+       "  <thead>\n",
+       "    <tr>\n",
+       "      <th id=\"T_6f30e_level0_col0\" class=\"col_heading level0 col0\" >Column Name</th>\n",
+       "      <th id=\"T_6f30e_level0_col1\" class=\"col_heading level0 col1\" >Type</th>\n",
+       "      <th id=\"T_6f30e_level0_col2\" class=\"col_heading level0 col2\" >Computed With</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td id=\"T_6f30e_row0_col0\" class=\"data row0 col0\" >image</td>\n",
+       "      <td id=\"T_6f30e_row0_col1\" class=\"data row0 col1\" >Image</td>\n",
+       "      <td id=\"T_6f30e_row0_col2\" class=\"data row0 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_6f30e_row1_col0\" class=\"data row1 col0\" >label</td>\n",
+       "      <td id=\"T_6f30e_row1_col1\" class=\"data row1 col1\" >String</td>\n",
+       "      <td id=\"T_6f30e_row1_col2\" class=\"data row1 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_6f30e_row2_col0\" class=\"data row2 col0\" >classification</td>\n",
+       "      <td id=\"T_6f30e_row2_col1\" class=\"data row2 col1\" >Json</td>\n",
+       "      <td id=\"T_6f30e_row2_col2\" class=\"data row2 col2\" >vit_for_image_classification(image, model_id='farleyknight-org-username/vit-base-mnist')</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n"
+      ],
+      "text/plain": [
+       "table 'mnist'\n",
+       "\n",
+       "   Column Name   Type                                                                            Computed With\n",
+       "         image  Image                                                                                         \n",
+       "         label String                                                                                         \n",
+       "classification   Json vit_for_image_classification(image, model_id='farleyknight-org-username/vit-base-mnist')"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "t"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9bfc6d8e-9a61-4399-8689-974caef19b24",
+   "metadata": {},
+   "source": [
+    "Pixeltable provides a range of operators on `Json`-typed output that behave just as you'd expect. To look up a key in a dictionary, use the syntax `t.classification['labels']`:"
```

Comment:
Worth mentioning that these are json path expressions, ie, we didn't just make this up randomly, and providing a link to something like
https://en.wikipedia.org/wiki/JSONPath